src/main/java/org/rug/args/ProjectArgsManager.java

```diff
@@ -18,7 +18,7 @@
     public File gitRepo;
 
     @Parameter(names = {"-javaProject", "-jP"}, description = "Flag this as a Java project (i.e. project analysed with Arcan for Java).")
-    public boolean isJava = true;
```

Comment:
Why did you change this?  
I'm asking because the analysis for Java is supposed to be the default one. If you change this here, the user has to use -jP every time he analyses a java project. 
Unless this is really necessary, I would suggest to undo this change.

Reply:
As discussed, this was messing with the C runner since the isJava flag was set to true by default and it would still be true when running  a C project. For the web API, the user has to set the `language` parameter anyway!